src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/components/FetchData.js

```diff
@@ -1,20 +1,35 @@
 import React, { Component } from 'react';
+////#if (IndividualLocalAuth)
+import authService from './api-authorization/AuthorizeService'
+////#endif
 
 export class FetchData extends Component {
   static displayName = FetchData.name;
 
-  constructor (props) {
+  constructor(props) {
     super(props);
     this.state = { forecasts: [], loading: true };
 
+    ////#if (IndividualLocalAuth)
+    authService.getAccessToken()
+      .then(token =>
+        fetch('api/SampleData/WeatherForecasts', {
+          headers: FetchData.getHeaders(token)
+        }))
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ forecasts: data, loading: false });
+      });
```

Comment:
Rather than all these `.then` callbacks, it would be really nice to be using `async`/`await`, partly for clarity and partly so people could more realistically add error handling if they wanted.

I know it's in a constructor, but it's currently fire-and-forget anyway, so this could be factored out into an `async` method and called from the constructor.

Reply:
Done